src/lang/type.ml

```diff
@@ -275,7 +291,8 @@ let repr ?(filter_out = fun _ -> false) ?(generalized = []) t : repr =
         | EVar (i, c) ->
             if List.exists (fun (j, _) -> j = i) g then uvar g t.level (i, c)
             else evar t.level i c
-        | Link t -> repr g t)
+        | Link (Covariant, t) when !debug -> `Misc ("[>", repr g t, "]")
```

Comment:
Do you think it'd be worthwhile to make this a setting? As the language gets more complex, we might need this info from users and a settings would make it easier to turn on.

Reply:
This is very very technical . For now, I only see this as useful for implementors of the type system.